metadata/nz_building_outlines_full_history.xml

```diff
@@ -104,50 +104,19 @@
 
-**Guidelines for Building Outline Extraction:**
-**The following rules were used for capturing buildings;**
-• Buildings under construction are not captured.
-Standalone dwellings:
-• Primary building structures are captured as separate polygons. Adjoining buildings are captured as a single polygon.
-• Extensions, sunrooms, balconies, patios, and annexes are captured as part of the primary building structure.
-• Permanent buildings structures such as sheds and greenhouses, not attached to the primary building structure, but larger than 10m² are captured as individual buildings.
-• Inner courtyards within buildings are captured as a hole within the building polygon if resolution of imagery allows. These holes should be no smaller than 10m².
-Townhouses:
-• Adjoining townhouses are captured a single polygon.
-Commercial:
-• Adjoining commercial buildings are captured as separate polygons with coincident shared edges.
-Rural/farmland areas:
-• Hay bales and silage are not captured.
-• Water tanks are captured as buildings. Minimum size is 16.5m².
-General:
-• Non-permanent objects such as shipping containers, caravans, large vehicles and trampolines should not be captured as buildings.
+Please refer to the [NZ Building Outlines Data Dictionary] (https://nz-buildings.readthedocs.io/en/latest/introduction.html) for detailed metadata and information about this dataset.
 
-**Spatial Accuracy:**
-Buildings must be;
-• Spatially accurate to 1 metre
-• Captured in full where they are partially occluded by vegetation or require additional viewer interpretation.
-• Squared off (90 degrees) if the corner angles are between 80-100 degrees.      
+This NZ Building Outlines Full History dataset contains all of the building outlines derived from multiple aerial imagery years in each region. 
 
-**Valid Geometry:**    
-A building outline polygon is considered to have valid geometry if;
-• It does not overlap with other building outline polygons;
-• It does not contain any spikes;
-• It does not contain lines that are intersected with each other;
-• It can contain polygons with interior rings (holes);
-• It does not contain polygons with multiple exterior rings;
-• Building polygons captured will be greater than 10m².
+The related [NZ Building Outlines] (https://data.linz.govt.nz/layer/xxxxx-nz-building-outlines) dataset contains only building outlines derived from the latest LINZ aerial imagery in each region.
```

Comment:
**More building outlines data**
[NZ Building Outlines Lifecycle] (https://data.linz.govt.nz/table/xxxxx) – stores the relationship between building outlines when one building is split into two or more building outlines, or when multiple building outlines are merged into a single outline. Use this in conjunction with the NZ Building Outlines Full History dataset.

[NZ Building Outlines] (https://data.linz.govt.nz/layer/xxxxx) - provides current building outlines only derived from the latest LINZ aerial imagery.